x-pack/plugins/fleet/public/components/enrollment_instructions/manual/platform_selector.tsx

```diff
@@ -28,14 +37,27 @@ const CommandCode = styled.pre({
 });
 
 export const PlatformSelector: React.FunctionComponent<Props> = ({
-  linuxMacCommand,
+  linuxCommand,
+  macCommand,
   windowsCommand,
-  installAgentLink,
+  linuxDebCommand,
+  linuxRpmCommand,
   troubleshootLink,
   isK8s,
 }) => {
   const { platform, setPlatform } = usePlatform();
 
+  const callout = (
```

Comment:
nitpick: maybe we can have a little more explicit variable name  something like`systemPackageCallout`?

```suggestion
  const systemPackageCallout = (
````